package/boot/uboot-envtools/files/mediatek

```diff
@@ -10,16 +10,39 @@ touch /etc/config/ubootenv
 . /lib/uboot-envtools.sh
 . /lib/functions.sh
 
+block_dev_path() {
+	local dev_path
+
+	case "$1" in
+	/dev/mmcblk*)
+		dev_path="$1"
+		;;
+	PARTLABEL=* | PARTUUID=*)
+		dev_path=$(blkid -t "$1" -o device)
+		[ -z "${dev_path}" -o $? -ne 0 ] && return 1
+		;;
+	*)
+		return 1;
+		;;
+	esac
+
+	echo "${dev_path}"
+	return 0
+}
+
 board=$(board_name)
 
 case "$board" in
+cmcc,rax3000m-emmc |\
+glinet,gl-mt6000 |\
+jdcloud,re-cp-03)
+	env_dev=$(block_dev_path "PARTLABEL=u-boot-env")
```

Comment:
why not use built-in function `find_mmc_part`?

Reply:
I wasn't aware of the find_mmc_part function. It would be better to use the built-in function find_mmc_part. Thank you for the suggestion.